### ISSUE TITLE
Not working on a live server

### ISSUE THREAD

**Issue by moyger** [fromtheoutfit/navee#31]
Hello, I tried to migrate my site to the live hosting but it seems that Navee suddenly have errors. It was working on 2 localhost. Is there an issue with site url or something?

**Comment by karloestrada** [comment]
Update:
I uploaded it again on the live server. I'm using mediatemple grid hosting. This is specifically the error: "Unexpected [ on Navee_NodeModel line 19" which is this one - 
'linkType'               => array(AttributeType::Enum, 'values' => [ 'entryId', 'assetId', 'categoryId', 'customUri', 'none' ]),

I changed to 'linkType'               => array(AttributeType::Enum, 'values' =>  'entryId', 'assetId', 'categoryId', 'customUri', 'none'  ),

Basically, removing the enclosing "[ ]". Weird but it works. Suddenly, I don't have errors. 

Maybe you guys can take a look....
